client/simple/theme_icons.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Generate icons.html for the jinja templates of the simple theme.
+ */
+
+import { argv } from "node:process";
+import { dirname, resolve } from "node:path";
+import { jinja_svg_sets } from "./tools/jinja_svg_catalog.js";
+
+const HERE = dirname(argv[1]) + "/";
+const dest = resolve(HERE, "../../searx/templates/simple/icons.html");
+
+/** @type import("./tools/jinja_svg_catalog.js").JinjaMacro[] */
+const searxng_jinja_macros = [
+  { name: "icon",       class: "sxng-icon-set" },
+  { name: "icon_small", class: "sxng-icon-set-small" },
+  { name: "icon_big",   class: "sxng-icon-set-big" },
+];
+
+
+const sxng_icon_opts ={
+  multipass: true,
+  plugins: [
+    { name: "removeTitle" },
+    { name: "removeXMLNS" },
+    { name: "addAttributesToSVGElement",
+      params: {
+        attributes: [
+          {
+            "aria-hidden": "true",
+          }]}}]
+};
+
+
+/** @type import("./tools/jinja_svg_catalog.js").IconSet */
+const simple_icons = [
+  {
+    base: resolve(HERE, "node_modules/ionicons/dist/svg"),
+    set: {
+      "alert": "alert-outline.svg",
+      "appstore": "apps-outline.svg",
+      "book": "book-outline.svg",
+      "close": "close-outline.svg",
+      "download": "download-outline.svg",
+      "ellipsis-vertical": "ellipsis-vertical-outline.svg",
+      "file-tray-full": "file-tray-full-outline.svg",
+      "film": "film-outline.svg",
+      "globe": "globe-outline.svg",
+      "heart": "heart-outline.svg",
+      "image": "image-outline.svg",
+      "layers": "layers-outline.svg",
+      "leecher": "arrow-down.svg",
+      "location": "location-outline.svg",
+      "magnet": "magnet-outline.svg",
+      "musical-notes": "musical-notes-outline.svg",
+      "navigate-down": "chevron-down-outline.svg",
+      "navigate-left": "chevron-back-outline.svg",
+      "navigate-right": "chevron-forward-outline.svg",
+      "navigate-up": "chevron-up-outline.svg",
+      "people": "people-outline.svg",
+      "play": "play-outline.svg",
+      "radio": "radio-outline.svg",
+      "save": "save-outline.svg",
+      "school": "school-outline.svg",
+      "search": "search-outline.svg",
+      "seeder": "swap-vertical.svg",
+      "settings": "settings-outline.svg",
+      "tv": "tv-outline.svg",
+    },
```

Comment:
@vojkovic: while we are just talking about icons .. this PR also fix some very old issues with some missing SVGs: e.g. SVGs for `seeder`, `lecher` and *file size* (`save`) had been missed in the past ..


![grafik](https://github.com/user-attachments/assets/dd9df0d0-c98e-4a7d-905d-1a2a13078385)


![grafik](https://github.com/user-attachments/assets/cb14ba57-949f-4e27-80f7-4f9166640e9d)

Reply:
Very nice - great work with this mate.